src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -35,14 +39,19 @@ public void SelectOutputFolder()
          _outputFolderDTO.FolderPath = outputFolder;
       }
 
+      public override void ViewChanged()
```

Comment:
Indeed, although I added that the button is disabled when the batch tool is running, so I changed it around a bit now.